src/socketify/loop.py

```diff
@@ -76,6 +78,16 @@ def set_timeout(self, timeout, callback, user_data):
     def create_future(self):
         return self.loop.create_future()
 
+    def start_uv_loop(self):
+        if not self.uv_thread_started:
+            t1 = threading.Thread(target=self.uv_loop.run, daemon=True)
+            t1.start()
+        self.uv_thread_started = True
```

Comment:
do we really need the thread? It adds some overhead (+1 thread /process) and this implementation leaks resources.